src/candela/VisComponent/mixin/AutoResize.js

```diff
@@ -0,0 +1,16 @@
+import Resize from './Resize';
+
+let AutoResize = Base => class extends Resize(Base) {
```

Comment:
I agree with (2). If there is reinitialization, we could file a separate optimization issue to be resolved later.